### ISSUE TITLE
Accessibility: MAS4.2.1: Voiceover is announcing as selected for unselected elements.

### ISSUE THREAD

**Issue by christychen126** [microsoft/vscode#97563]
Build details
Mac:
macOS Catalina: 10.15.2
VS code version: 1.43.1

Repro Steps:​
1. Launch VS code application
2. Navigate to 'Elements for 'Microsoft edge'(or any extension) button and observe voiceover announcement
 
Note: Issue reproes for all the buttons in the application
 
Expected Result:
-> On navigation, voiceover should announce as selected for selected and unselected for unselected buttons.
-> Selected state should be defined for only required buttons 

Actual Result:
-> On navigating to 'Elements for 'Microsoft edge' button, voiceover is announcing as selected for both selected and unselected buttons.
-> Selected state is defined for all the buttons in the application

User Impact:
User with visual impairment will face difficulty as they will think that the button is already selected

MAS Reference Link: MAS 4.2.1 - Object Information
![image](https://user-images.githubusercontent.com/20508096/81625691-78e3a980-93ae-11ea-894e-781f4fc351ee.png)
![image](https://user-images.githubusercontent.com/20508096/81625711-87ca5c00-93ae-11ea-9a2f-4ffd500c70af.png)

**Comment by isidorn** [comment]
You are using vscode 1.43 that is an old version.
I can not reproduce this with latest vscode insdiers
1.46
Please try it out with vscode insiders and if you can reproduce let us know

![recording](https://user-images.githubusercontent.com/1926584/81657189-f8698b00-9437-11ea-8d52-5056638471fa.gif)

**Comment by christychen126** [comment]
Gave 1.46 a try and the issue seems to persists. When tabbing to the left toolbar it announced the button at selected (without hitting return key). I attached version details and a gif. 
![ally](https://user-images.githubusercontent.com/20508096/81727530-f73b5c80-943d-11ea-8e34-75e431903964.gif)
![image](https://user-images.githubusercontent.com/20508096/81727827-65801f00-943e-11ea-8acf-00c738fcf462.png)

**Comment by isidorn** [comment]
@christychen126 I believe this is up to VoiceOver and the verbosity level you have set there.
We do not set any aria-selected on the buttons.
@marcozehe when does VoiceOver say "selected"? We do not set any funky aria attribute on our activity bar icons. They have the role `button` and belong to role `toolbar`. Easy repro: F6 a couple of times to focus on activity bar, up / down to navigate.

**Comment by MarcoZehe** [comment]
@isidorn There is that thing where one of the items like Explorer or Extensions, is the active panel, right? That is being exposed with the aria-selected attribute I believe. But aren‘t these tabs, not buttons? I am not in front of my computer right now, so cannot check. In that case, VoiceOver would very well say „selected“.

**Comment by isidorn** [comment]
@marcozehe yeah, those are the "Explorer" and "Extensions" things. They are sort of like tabs, visually they do not look that way though, since they are all on the vertical line all the way to the left one above the other. Clicking on them opens different views in the sidebar.
It is not urgent, if you have time just check them out tomorrow and let me know what you think. Thanks :)

Though I see that we use role 'tab' for the exactly same things in the bottom panel (which you get via Cmd + J). So maybe we should use role `tab` here as well to be consistent.

**Comment by isidorn** [comment]
Let's reopen this.
@marcozehe friendly ping. I plan to use role `tab` in the activity bar

**Comment by isidorn** [comment]
After some reading I decided to go with the `role = tab` for activity bar items and to use `aria-expanded` on each of the elements, so the active and visible one would have `aria-expanded = true`, all else would have `aria-expanded false`
